widget_bandsplot/schemas/pdos.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$id": "https://example.com/pdos",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "JSON Schema for an PDOS entry",
+    "type": "object",
+    "required": ["dos"],
+    "properties": {
+        "fermi_energy": {"type": "number"},
+        "dos": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "required": ["label", "x", "y"],
+                "properties": {
+                    "label": {"type": "string"},
+                    "x": {
+                        "type": "array",
+                        "items": {"type": "number"}
+                    },
+                    "y": {
+                        "type": "array",
+                        "items": {"type": "number"}
+                    },
+                    "lineStyle": {"enum": ["solid", "dash"]},
+                    "borderColor": {"type": "string"},
+                    "backgroundColor": {"type": "string"}
```

Comment:
Should we maybe place all representation-related (i.e. not data) entries in a sub-property?

Reply:
I think this format reflects what matplotlib did for plotting lines. and it is simpler at the same level for all parameters.

Reply:
Sure.